js/cloze.js

```diff
@@ -39,8 +39,17 @@
       if (behaviour.caseSensitive !== true) {
         answered = answered.toLowerCase();
       }
-
       for (var i = 0; i < answers.length; i++) {
+        // Damerau-Levenshtein comparison
+        if (behaviour.acceptSpellingErrors === true) {
```

Comment:
I added a JSDoc comment and a general comment in the code, because I want to change the correctness computation and probably would forget to change the JSDoc comment :-)